README.md

```diff
@@ -136,3 +136,57 @@ Find a bug? Contact us via [support.newrelic.com](http://support.newrelic.com/),
 or email support@newrelic.com.
 
 New Relic, Inc.
+
+
+## FAQs
+
+###SDK related questions:
```

Comment:
Nitpick: add a space after `###` since Github doesn't render this line as a header:
![image](https://user-images.githubusercontent.com/939550/40720771-1de7cc90-6418-11e8-8798-bf0ced4c9448.png)